CHANGES.rst

```diff
@@ -96,6 +96,16 @@
   verify as ``IFullMapping``, ``ISequence`` and ``IReadSequence,``
   respectively on all versions of Python.
 
+- Add human-readable ``__str___`` and ``__repr___`` to ``Attribute``
+  and ``Method``. These contain the name of the defining interface
+  and the attribute. For methods, it also includes the signature.
```

Comment:
I would add here: "This may break tests in consuming code - in special doctests - depending on the exact output of the methods. Those tests need to be adapted."

Reply:
Will do!